app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -332,6 +334,26 @@ class RawSubmissionTable extends React.Component {
     }, after_function);
   };
 
+  create_collect_submissions_channel_subscription(submissionTable) {
```

Comment:
Oh sorry I don't think I was clear, you shouldn't need to pass in `this` here. Since you've made this function a method of the class, you can use `this` directly.

However, you should use the "class properties" syntax for defining a method: `create_collect_submissions_channel_subscription = () => { ...`